addon/templates/components/admin-fields/admin-field-boolean.hbs

```diff
@@ -0,0 +1,10 @@
+{{#unless hasInitialValue}}
+  <option value selected disabled>
+    {{placeholderText}}
+  </option>
+{{/unless}}
+{{#each data as |bool|}}
+  <option value={{bool}} selected={{eq bool boolValue}} data-test={{concat 'admin-field-boolean-' bool}}>
```

Comment:
If I understand this correctly, this will only ever have to values, `true` and `false` correct? Wouldn't a checkbox be better suited for this than a dropdown?